client/core/core.go

```diff
@@ -3516,7 +3516,7 @@ func handleRevokeMatchMsg(c *Core, dc *dexConnection, msg *msgjson.Message) erro
 
 	tracker, _, _ := dc.findOrder(oid)
 	if tracker == nil {
-		return fmt.Errorf("no order found with id %s", oid.String())
+		return fmt.Errorf("no order found with id %s (not an error if you've completed your side of the swap)", oid.String())
```

Comment:
Here's a possible TODO for `Swapper`.  As maker, client redeemed and got an ack from their 'redeem', so they retired the trade, but if the server is still in `MakerRedeemed` because taker fails to send their redeem and then it gets revoked, the maker's client don't find the order for the revoked match because they already completed and retired it.  Server could not send revoke_match to the maker when the match is `MakerRedeemed`.